swiggy-pdp-cart/src/components/Main/Content/Content.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import Menu from "./Menu/Menu";
+import Cart from "./Cart/Cart";
+import Categories from "./Categories/Categories";
+import { fetchCategoriesList,fetchMenuList } from "../../../services/fakeFetchFunctions";
+import { categoryNameById } from "./helper/helper";
+import ContentCSS from './content.module.css'
+export default function Content(){
+  
+  const [category,setCategory]=useState('recommended');
+  const [menu,setMenu]=useState([]);
+  const [categoryList,setCategoryList]=useState([]);
+  
+  useEffect(() => {
+    const menu = fetchMenuList();
+    const categoryList = fetchCategoriesList();
+    setMenu(menu);
+    setCategoryList(categoryList);
+  },[]);  
+  const menuList = menu.filter((dish) => dish.categories.includes(category));
+  const menuHeading = categoryNameById(category, categoryList);
```

Comment:
This function is incorrectly named.

`categoryNameById` is a noun while the function names are supposed to be verbs as they denote an action.

Rename this method to denote that it is returning a heading.